core/src/test/java/nav/statusplattform/core/repositories/OpeningHoursRepositoryTest.java

```diff
@@ -388,17 +421,134 @@ void removeOpeningHoursFromService() {
     void getOHGroupForService(){
         //Arrange
         /*Create service*/
-        ServiceEntity service = SampleData.getRandomizedServiceEntity();
+        ServiceEntity service = getRandomizedServiceEntity();
         UUID serviceId = serviceRepository.save(service);
         /*Create group*/
-        OpeningHoursGroupEntity group = SampleData.getRandomizedOpeningHoursGroupEntity();
+        OpeningHoursGroupEntity group = getRandomizedOpeningHoursGroupEntity();
         group.setId(openingHoursRepository.saveGroup(group));
         UUID groupId = group.getId();
         openingHoursRepository.setOpeningHoursToService(groupId, serviceId);
         //Act
         Optional<OpeningHoursGroup>retrievedGroup = openingHoursRepository.getOHGroupForService(serviceId);
         //Assert
+        Assertions.assertThat(retrievedGroup).isPresent();
         Assertions.assertThat(retrievedGroup.get().getId()).isEqualTo(group.getId());
     }
 
+    private OpeningHoursRuleEntity getRandomizedOpeningRule() {
+        return new OpeningHoursRuleEntity()
+                .setName(getRandomFromArray(areaNames))
+                .setRule(getRandomFromArray(rules));
+    }
+
+    private String getRandomFromArray(ArrayList<String> array) {
+        //Hit skal man ikke komme
+        if (array.isEmpty()) return null;
+        Random random = new Random();
+        return array.get(random.nextInt(array.size()));
+    }
```

Comment:
I suggest you keep code like this gathered in one common location. I don't consider this code special, at least wrt to openinghours.

Reply:
Thanks for the tips Christian. I have only made changes to this code before your message. Its now apart of SampleData.